packages/airnode-adapter/tsconfig.build.json

```diff
@@ -1,10 +1,7 @@
 {
-  "extends": "../../tsconfig.build.json",
-  "compilerOptions": {
-    "baseUrl": "./",
-    "outDir": "./dist",
-    "rootDir": "src"
-  },
-  "include": ["src/**/*"],
-  "exclude": ["src/**/*.test.ts"]
+  "extends": "./tsconfig.json",
+  // References are not inhereted and need to be listed explicitly. See:
```

Comment:
```suggestion
  // References are not inherited and need to be listed explicitly. See:
```

Reply:
Not sure why my grammar corrector didn't catch this...